### DISCUSSION TITLE
How can I link multiple contacts?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#16167] [How to]
Hi guys,

I would like to know how I can link multiple contacts each other.

For example, I have multiple contacts that they are members of the same family and I would like to tag them. So, when I open one of these contacts, I can see all the family members attached to this contact.

How could i do it? :-)

Regards,

**Comment by mariuszkrzaczkowski** [comment]
The easiest way to add an additional relationship is Family M2M (Many to many)

**Reply by mariuszkrzaczkowski** [reply]
![image](https://user-images.githubusercontent.com/10198654/177095146-c94a7aaa-9b07-4f7e-8b65-3db2673c2775.png)

**Comment by OscarGS86** [comment]
Thanks @mariuszkrzaczkowski ,

Please, could you let me know how I can add an additional relationship Family as M2M? Should I create a new field for contacts?

Thanks for your help.

**Comment by OscarGS86** [comment]
HI @mariuszkrzaczkowski ,

I cannot see the "Add Relation" blue button on that section. Do you know how I can enable it?

I am Super User and I still cannot see it.

Thanks for your help.
![Screenshot 2022-07-17 at 07 33 45](https://user-images.githubusercontent.com/92228499/179385410-13c224d8-5605-4340-a3ad-f65b4d24411f.png)

**Reply by vovpff** [reply]
Check for this option config/Developer.php

**Comment by mariuszkrzaczkowski** [comment]
https://doc.yetiforce.com/code/classes/Config-Developer.html#property_CHANGE_RELATIONS
